pycbc/io/live.py

```diff
@@ -446,25 +441,15 @@ def upload(self, fname, gracedb_server=None, testing=True,
             snr_series_plot_fname = self.basename + '_snr.png'
             asd_series_plot_fname = self.basename + '_asd.png'
 
-            pl.figure()
+            triggers = {
+                ifo: (self.coinc_results[f'foreground/{ifo}/end_time']
+                      + self.time_offset,
+                      self.coinc_results[f'foreground/{ifo}/snr'])
+                for ifo in self.et_ifos
+                }
             ref_time = int(self.merger_time)
-            for ifo in sorted(self.snr_series):
-                curr_snrs = self.snr_series[ifo]
-                curr_snrs.save(snr_series_fname, group='%s/snr' % ifo)
```

Comment:
After this change, PyCBC Live will no longer save the SNR timeseries to an HDF file. Is this intended? Maybe it is, as I do not remember actually using such files ever. However, if this is a conscious decision, then we can probably drop the `snr_series_fname` variable?

Reply:
…and do not forget that the HDF file is actually uploaded to GraceDB a few lines below! I thought we had removed this…

Reply:
Nope, it is still there!

I am not super happy with a PR for the offline search modifying the behavior of the online search. If we want to drop the HDF file, can we at least do so in a separate mini-PR, for a simpler history?

Reply:
Re-instated this for this PR, we can think about dropping this somewhere else

Reply:
(This was a mistake more than an intentional change!)